### DISCUSSION TITLE
Manually specifying many-to-many "through" table

### DISCUSSION THREAD

**Discussion by IvanDrag0** [nocodb/nocodb#6880] [Q&A]
Currently (from my understanding), whenever you create a Many-to-Many relationship, NocoDB uses an internal "through/joiner" table. However, that table is located in the NocoDB database and is not reflected in the main database that is being edited (which means that it's lost if I were to look at the table from another tool). Is it possible to manually specify the "through" table for those types of connections (so that they're also shown in the table's ERD diagram)?

**Comment by ReedyBear** [comment]
Did you find an answer to this?

On my system, it makes those joiner tables in my db (mysql), but they're named something like `nc_gked___nc_m2m_1_1wugki89` which is not good for working directly with my db.
